src/Estimators/EstimatorManagerNew.h

```diff
@@ -56,6 +57,18 @@ class EstimatorManagerNew
   EstimatorManagerNew(const QMCHamiltonian& ham, Communicate* comm);
```

Comment:
Not to be addressed in this PR. This constructor seems being used in test only. I'm wondering if we should change all the tests and delete this constructor.